subprojects/docs/src/docs/release/notes.md

```diff
@@ -272,6 +272,33 @@ There are better ways for re-using task logic, for example by using [task depend
 
 The default output location in [EclipseClasspath](dsl/org.gradle.plugins.ide.eclipse.model.EclipseClasspath.html#org.gradle.plugins.ide.eclipse.model.EclipseClasspath:defaultOutputDir) changed from `${project.projectDir}/bin` to `${project.projectDir}/bin/default`.
 
+### Blacklist repository on `IOException`s 
+
+In the event of a timeout (or any `IOException`s which are not caused by error HTTP status code), Gradle will skip subsequent connections to the same repository for the duration of the build. 
+The output of a build clearly indicates which request was skipped.
+
+```
+* What went wrong:
+Could not resolve all files for configuration ':deps'.
+> Could not resolve group:a:1.0.
+  Required by:
+      project :
+   > Could not resolve group:a:1.0.
+      > Could not get resource 'http://localhost:54347/repo/group/a/1.0/a-1.0.pom'.
+         > Could not GET 'http://localhost:54347/repo/group/a/1.0/a-1.0.pom'.
+            > Read timed out
+> Could not resolve group:b:1.0.
+  Required by:
+      project :
+   > Skipped due to earlier error
+```
+
+### Dependency resolution order
```

Comment:
:x: I'd change this title to be more clear e.g. "Failure to resolve dependency from repository avoids checking other declared repositories".

Reply:
Changed to `Avoid checking other repositories when dependency resolution in one repository fails`, would it be better?